gcloud-java-pubsub/baseline/src/main/java/com/google/gcloud/pubsub/spi/v1/PublisherSettings.java

```diff
@@ -142,45 +141,54 @@
   private final MethodBuilders methods;
 
   private static class MethodBuilders {
-    private final ApiCallableBuilder<Topic, Topic> createTopicMethod;
-    private final BundlableApiCallableBuilder<PublishRequest, PublishResponse> publishMethod;
-    private final ApiCallableBuilder<GetTopicRequest, Topic> getTopicMethod;
-    private final PageStreamingApiCallableBuilder<ListTopicsRequest, ListTopicsResponse, Topic>
+    private final ApiCallable.Builder<Topic, Topic> createTopicMethod;
+    private final ApiCallable.BundlableBuilder<PublishRequest, PublishResponse> publishMethod;
+    private final ApiCallable.Builder<GetTopicRequest, Topic> getTopicMethod;
+    private final ApiCallable.PageStreamingBuilder<ListTopicsRequest, ListTopicsResponse, Topic>
         listTopicsMethod;
-    private final PageStreamingApiCallableBuilder<
+    private final ApiCallable.PageStreamingBuilder<
             ListTopicSubscriptionsRequest, ListTopicSubscriptionsResponse, String>
         listTopicSubscriptionsMethod;
-    private final ApiCallableBuilder<DeleteTopicRequest, Empty> deleteTopicMethod;
+    private final ApiCallable.Builder<DeleteTopicRequest, Empty> deleteTopicMethod;
     private final ImmutableList<? extends ApiCallSettings> allMethods;
 
     public MethodBuilders() {
-      createTopicMethod = new ApiCallableBuilder<>(PublisherGrpc.METHOD_CREATE_TOPIC);
+      createTopicMethod = new Builder<>(PublisherGrpc.METHOD_CREATE_TOPIC);
       createTopicMethod.setRetryableCodes(RETRYABLE_CODE_DEFINITIONS.get("idempotent"));
       createTopicMethod.setRetryParams(RETRY_PARAM_DEFINITIONS.get("default"));
 
+      BundlingSettings publishBundlingSettings =
```

Comment:
I question the value of the bundling for createTopic method. If any, I would say getTopic is more likely to benefit from it.